source/features/hide-obvious-tooltips.css

```diff
@@ -1,15 +1,12 @@
 /* Remove tooltips where unnecessary */
-[aria-label^='View the whole file']::before,
-[aria-label^='View the whole file']::after,
-[aria-label^='Change this file']::before,
-[aria-label^='Change this file']::after,
-[aria-label='Delete this file']::before,
-[aria-label='Delete this file']::after,
-[aria-label='Edit this file']::before,
-[aria-label='Edit this file']::after,
-[aria-label='Edit comment']::before,
-[aria-label='Edit comment']::after,
-.notification-indicator::before,
-.notification-indicator::after {
+:is(
+[aria-label^='View the whole file'],
+[aria-label^='Change this file'],
+[aria-label='Delete this file'],
+[aria-label='Edit this file'],
+[aria-label='Edit comment'],
+[aria-label='Edit comment'],
+.notification-indicator
```

Comment:
Indentation?

Reply:
Why I put it on a new line? or why did I not add a tab?

Reply:
```suggestion
	  [aria-label^='View the whole file'],
	  [aria-label^='Change this file'],
	  [aria-label='Delete this file'],
	  [aria-label='Edit this file'],
	  [aria-label='Edit comment'],
	  [aria-label='Edit comment'],
	  .notification-indicator
```

Reply:
Linter does not let

Reply:
See https://github.com/sindresorhus/refined-github/runs/1999791363

Reply:
Looks like a stylelint problem. Content inside parens should be indented

Reply:
https://github.com/stylelint/stylelint/issues/5181